packages/core/src/stepper/internal/StepScreenReaderOnly.css

```diff
@@ -1,4 +1,4 @@
-.saltStepSROnly {
+.saltStepScreenReaderOnly {
```

Comment:
Have we confirmed these 4 rules is the best option?  Compare with `.salt-visuallyHidden`